scripts/core/editor3/reducers/toolbar.jsx

```diff
@@ -223,6 +229,38 @@ const applyEmbed = (state, code) => {
     return onChange(state, editorState);
 };
 
+
+/**
+ * @ngdoc method
+ * @name toggleParagraphMark
+ * @param {Object} state
+ * @return {Object} returns new state
+ * @description Enable/Disable the paragraph marks
+ */
+const toggleParagraphMark = (state) => {
```

Comment:
I think it would toggle it off when you toggle spellchecker after activating this

Reply:
@petrjasek hmmm it does... why is that? 🤔

Reply:
because [there it's also changing decorators](https://github.com/superdesk/superdesk-client-core/blob/master/scripts/core/editor3/reducers/spellchecker.jsx#L61) but it has no idea about this new one, it should be probably somehow combined together